src/base-element.js

```diff
@@ -503,8 +530,8 @@ export class BaseElement {
    * @param {!Element|!Array<!Element>} elements
    * @protected
    */
-  schedulePause(elements) {
-    this.resources_.schedulePause(this.element, elements);
+  scheduleUnload(elements) {
```

Comment:
Certain elements can't be paused (twitter videos), so we do have to unload to stop their playback.